common/scala/src/main/scala/whisk/core/containerpool/Container.scala

```diff
@@ -48,13 +48,24 @@ import whisk.core.entity.ActivationEntityLimit
  * for different container providers, but the implementation also needs to include
  * OpenWhisk specific behavior, especially for initialize and run.
  */
-case class ContainerId(val asString: String) {
+case class ContainerId(asString: String) {
   require(asString.nonEmpty, "ContainerId must not be empty")
 }
-case class ContainerAddress(val host: String, val port: Int = 8080) {
+case class ContainerAddress(host: String, port: Int = 8080) {
   require(host.nonEmpty, "ContainerIp must not be empty")
 }
 
+object Container {
+
+  /**
+   * The action proxies insert this line in the logs at the end of each activation for stdout/stderr
```

Comment:
small nit, can you end this sentence with a `.`
then squash all the commits to clean up the commit message.